recipes/bioconductor-ballgown/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: bioconductor-ballgown
+  version: 2.2.0
+source:
+  fn: ballgown_2.2.0.tar.gz
+  url: http://bioconductor.org/packages/3.2/bioc/src/contrib/ballgown_2.2.0.tar.gz
+  md5: 717570395edef570a4606858bafc6edb
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+requirements:
+  build:
+    - bioconductor-biobase
+    - bioconductor-biocgenerics
```

Comment:
The best practice is to only pin versions if you compile against a library, like boost.
If you want to have strict reproducible, you can and should use the requirements.txt file. Essentially you install ballgown and create a requirements.txt file from this. Save it and reuse it later, this will always give you the same dependencies.